samples/led-animate/AnimateLeds.cs

```diff
@@ -0,0 +1,167 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Device.Gpio;
+using System.Threading;
+using System.Linq;
+using System.Collections.Generic;
+using Iot.Device.Multiplexing;
+
+public class AnimateLeds : IDisposable
+{
+    private readonly int[] _leds;
+    private readonly int[] _ledsRev;
+    private IOutputSegment _segment;
+    public int LitTimeDefault = 200;
+    public int DimTimeDefault = 50;
+    public int LitTime = 200;
+    public int DimTime = 50;
```

Comment:
Public non-const fields? Is this really intended? Prefer either constants or properties (if changing should be possible).

Reply:
Fixed